requirements.txt

```diff
@@ -0,0 +1,3 @@
+python-terraform<=0.10.1
```

Comment:
why the `<=` instead of `==`? dependabot will ignore this `<=` notation...

Reply:
didn't realize dependabot was choosy regarding these. updated.